srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -3313,7 +3361,7 @@ CONFIG_MT76x0_COMMON=m
 CONFIG_MT76x0U=m
 CONFIG_MT76x0E=m
 CONFIG_MT76x2_COMMON=m
-CONFIG_MT76x2E=m
+# CONFIG_MT76x2E is not set
```

Comment:
You are disabling lots of options rather than building them as modules

Reply:
was not enabled since 4.16

Reply:
but imho even if it increases its size, what will have to do if X guy appears and says that he wants to use it? build it as a module, that's my point.